cluster-service/deploy/templates/deployment.yaml

```
@@ -136,6 +136,13 @@ spec:
         - name: azure-operators-managed-identities-config
           mountPath: /configs/azure-operators-managed-identities-config.yaml
           subPath: azure-operators-managed-identities-config.yaml
+        {{- if .Values.tracing.enabled }}
+        env:
+        - name: OTEL_EXPORTER_OTLP_ENDPOINT
+          value: "{{ .Values.tracing.address }}"
+        - name: OTEL_TRACES_EXPORTER
+          value: "{{ .Values.tracing.exporter }}"
```

```suggestion
          value: "otlp"
```

Perhaps we want to put the value `otlp` here constantly?

---

Do we any other type of exporter besides `otlp`?